src/OpenTK/Platform/Windows/WinGLNative.cs

```diff
@@ -1195,68 +1309,18 @@ public override MouseCursor Cursor
-                        Functions.SetCursor(cursor_handle);
                     }
                     else
                     {
-                        var stride = value.Width *
-                            (Bitmap.GetPixelFormatSize(PixelFormat.Format32bppArgb) / 8);
-
-                        Bitmap bmp;
-                        unsafe
-                        {
-                            fixed (byte* pixels = value.Data)
-                            {
-                                bmp = new Bitmap(value.Width, value.Height, stride,
-                                    PixelFormat.Format32bppArgb,
-                                    new IntPtr(pixels));
-                            }
-                        }
-                        using (bmp)
-                        {
-                            var iconInfo = new IconInfo();
-                            var bmpIcon = bmp.GetHicon();
-                            var success = Functions.GetIconInfo(bmpIcon, out iconInfo);
-
-                            try
-                            {
-                                if (!success)
-                                {
-                                    throw new System.ComponentModel.Win32Exception();
-                                }
-
-                                iconInfo.xHotspot = value.X;
-                                iconInfo.yHotspot = value.Y;
-                                iconInfo.fIcon = false;
-
-                                var icon = Functions.CreateIconIndirect(ref iconInfo);
-                                if (icon == IntPtr.Zero)
-                                {
-                                    throw new System.ComponentModel.Win32Exception();
-                                }
-
-                                // Need to destroy this icon when/if it's replaced by another cursor.
-                                cursor = value;
-                                cursor_handle = icon;
-                                Functions.SetCursor(icon);
-                            }
-                            finally
-                            {
-                                if (success)
-                                {
-                                    // GetIconInfo creates bitmaps for the hbmMask and hbmColor members of ICONINFO. 
-                                    // The calling application must manage these bitmaps and delete them when they are no longer necessary.
-                                    Functions.DeleteObject(iconInfo.hbmColor);
-                                    Functions.DeleteObject(iconInfo.hbmMask);
-                                }
-
-                                Functions.DestroyIcon(bmpIcon);
-                            }
-                        }
+                        var data = CreateAniFromCursor(value);
+                        cursor_handle = Functions.CreateIconFromResource(data, data.Length, false, 0x00030000);
```

Comment:
Some magic from MS: https://msdn.microsoft.com/de-de/library/windows/desktop/ms648060(v=vs.85).aspx

Best I could do is comment it as "Version that is generally set to 0x00030000" referrring to above url.